docs_source_files/content/remote_webdriver/remote_webdriver_client.de.md

```diff
@@ -87,7 +98,25 @@ caps.version = 67
 driver = Selenium::WebDriver.for :remote, :url => "http://www.example.com", :desired_capabilities => caps
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR  
+const { Builder, Capabilities } = require("selenium-webdriver");
+const chrome = require("selenium-webdriver/chrome")
+var capabilities = Capabilities.chrome();
+//To avoid InsecureCertificateError for selenium4-aplha5
+capabilities.setAcceptInsecureCerts(true);
+const width = 780;
+const height = 480;
+(async function helloSelenium() {
+    let driver = new Builder()
+        //Browser Window Size Set
+        .setChromeOptions(new chrome.Options().windowSize({ width, height }))
```

Comment:
I am really sorry, perhaps I have not been clear 😢 
The other examples are living through the idea of: `suppose you wanted to run Chrome on Windows XP using Chrome version 67`...
So, if we see the Java example, it is using ChromeOptions to set those values. It would be nice to see the JavaScript example do the same thing, to keep uniformity. We do not need to set a window size or headless option, it is to make all the examples in different languages do the same.
Does it make sense?